<?xml version="1.0"?>
<launch>
    <include file="$(find openni_launch)/launch/openni.launch">
        <arg name="camera" value="camera" />
    </include>

    <node name="image_view_kinect" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/openni/rgb/image_color"/>
        <param name="autosize" value="false" />
    </node>

    <node name="disparity_view" pkg="image_view" type="disparity_view" respawn="false" output="screen">
        <remap from="image" to="/openni/depth_registered/disparity"/>
        <param name="autosize" value="false" />
    </node>
</launch>

